### ISSUE TITLE
Speed Issues with smtp mail

### ISSUE THREAD

**Issue by bobi33** [kjac/FormEditor#191]
Hi @kjac, 

If you remember my previous thread comments regarding slow form submissions using sync...

-I am now using the latest version of umbraco v 7.10.4, and the latest nuget install of formeditor 
-I have tested both send grid and mailgun...both form submissions under sync are extremely slow
-when I remove send grid or mailgun from the mail settings, the form loads perfectly fine after a submission
-the html email sent is not lengthy at all...it's basically your template minus all the words except: 'you have received a new email from your website'
-SendGrid SMTP works perfectly fine with other notifications (i.e. Umbraco password reset); FormEditor works perfectly fine without setting up the SMTP SendGrid; but form submission takes forever as soon as I add the SMTP SendGrid info
-even though the form is loading upon pressing submit, the actual form details (the message) is uploaded in the back-office before the form fully loads/ shows a complete submission to the user (about not even 10 seconds after clicking submit, the data shows up in the back office, but takes 40-60 seconds to load the form fully for the user indicating a successful submission)

Can this be related to a folder permission issue in IIS (I tried playing around with full access permissions...still no luck though). 

I can give you the link to the form so you can see how slow it is...again, it's truly strange that the umbraco back end forgot password works really fast with send grid ... unlike the form editor...

I have done the following in regard to smtp email settings in web.config, umbracosettings.config, iis, and under the form tab in the back office:
-in web.config I inputed "from" email address under mail settings as my gmail address, and I configured the send grid network credentials
-in umbracosettings.config I changed the notifications email to the same gmail address as above
-in the back office in the form tab in the back office I put the recipient address as a customized outlook address, and the sender email address as the above gmail address.
-in the back office under document types, I added the smtp information to the form tab / under the data type for form editor in the developer section.

Any ideas?

**Comment by kjac** [comment]
Hi @bobi33,

I do remember the issue. I haven't experienced any similar issues, and I can't reproduce the issue either. 

One of the really awesome things about Umbraco is that it's so easy to get up and running using the NuGet package and SQLCE. I suggest you create a brand new web project and install Umbraco and Form Editor from NuGet, run it off localhost and see if you can reproduce the issue with a barebone installation.

**Comment by bobi33** [comment]
Hmm, ok, I've determined the cause of the issue...Sendgrid via smtp and mailgun via smtp just work slow with this. I tried mail trap (per your demonstration), and that works faster.

If I were to integrate a web service instead of an smtp, what would the host name become? In smtp in mail trap, the host name / server name is 	smtp.mailtrap.io, but if I were to make my own web service, what would that become?
